sfja/src/index.js

```diff
@@ -34,8 +36,9 @@ const routing = (
       <Route exact path="/">
         <Redirect to="/dashboard" />
       </Route>
-      {/* <Route exact path="/admin" component={Admin} />*/}
-      <Route path="/form/:key" component={Form} />
+      <Route path={['/parentdash/:parentKey/:studentId', '/parentdash/:parentKey/']} component={ParentHeader} />
```

Comment:
@Antranduc take a look at this -- the header renders regardless and can take in the key if it wants.
I think I forgot a null checker for this.props.params.match.studentId but I think it works fine as-is now.